pkg/endpoints/pipeline_test.go

```diff
@@ -1145,3 +1085,245 @@ func TestPipelineRunUpdateStatusAlreadySet412(t *testing.T) {
 		t.Errorf("FAIL: should have received a http 412 code when setting the status to something already set, got %d", resp.StatusCode())
 	}
 }
+
+/* PipelineRun create bad request 400 */
+func TestCreatePipelineRunBadRequest(t *testing.T) {
+
+	r := dummyResource()
+
+	pipeline1 := v1alpha1.Pipeline{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "Pipeline1",
+		},
+		Spec: v1alpha1.PipelineSpec{},
+	}
+
+	_, err := r.PipelineClient.TektonV1alpha1().Pipelines("ns1").Create(&pipeline1)
+	if err != nil {
+		t.Errorf("FAIL: error creating test pipeline: %s", err)
+	}
```

Comment:
Please make this into a function:
```
pipeline1 := v1alpha1.Pipeline{
	ObjectMeta: metav1.ObjectMeta{
		Name: "Pipeline1",
	},
	Spec: v1alpha1.PipelineSpec{},
}

_, err := r.PipelineClient.TektonV1alpha1().Pipelines("ns1").Create(&pipeline1)
if err != nil {
	t.Errorf("FAIL: error creating test pipeline: %s", err)
}
```
It is repeated almost every time in each test function.